sw/device/lib/dif/dif_hmac_unittest.cc

```diff
@@ -32,7 +32,7 @@ class HmacTest : public testing::Test, public mock_mmio::MmioTest {
   struct ConfigRegister {
     bool hmac_enable = false;
     bool sha_enable = true;
-    bool msg_little_endian = true;
+    bool msg_big_endian = false;
```

Comment:
@doreis-lowrisc I rename the field to `big_endian` as the meaning of the field was reverted in this PR.